### ISSUE TITLE
WebSocketService receives event but ChatEventHanlder doesn't receive event

### ISSUE THREAD

**Issue by savked** [GetStream/stream-chat-android#241]
I am trying to handle notifications received while the app is in foreground.

1. Testing on two devices, one is in ChatFragment (device 1) and one is in MainActivity waiting for notification (device 2).
2. Sending a message to device 2
3. Device 1 receives a "notification" from WebSocket service that I see printed in logcat like this:
```
2019-10-25 13:20:49.238 15480-15613/de.tourlina.Tourlina.staging D/WebSocketService: WebSocket # 1 Response : {"cid":"messaging:181-182","type":"notification.message_new","message":{"id":"182-e63146e0-03c6-4b26-a97f-37386da963ca","text":"gujfjf","html":"\u003cp\u003egujfjf\u003c/p\u003e\n","type":"regular","user":{"id":"182","role":"user","created_at":"2019-09-17T10:14:45.713314Z","updated_at":"2019-10-25T11:14:01.658747Z","last_active":"2019-10-25T11:20:32.796786818Z","online":true,"name":"Ivana"},"attachments":[],"latest_reactions":[],"own_reactions":[],"reaction_counts":null,"reply_count":0,"created_at":"2019-10-25T11:20:44.307975Z","updated_at":"2019-10-25T11:20:44.307976Z","mentioned_users":[]},"channel":{"id":"181-182","type":"messaging","cid":"messaging:181-182","last_message_at":"2019-10-25T11:20:44.307975Z","created_at":"2019-10-02T08:51:31.252514Z","updated_at":"2019-10-02T08:51:31.252514Z","created_by":null,"frozen":false,"members":[{"user_id":"181","user":{"id":"181","role":"user","created_at":"2019-09-17T10:13:45.485842Z","updated_at":"2019-10-25T11:20:28.772961Z","last_active":"2019-10-25T11:20:29.872239224Z","online":true,"name":"Petar"},"created_at":"2019-10-02T08:51:31.254688Z","updated_at":"2019-10-02T08:51:31.254688Z"},{"user_id":"182","user":{"id":"182","role":"user","created_at":"2019-09-17T10:14:45.713314Z","updated_at":"2019-10-25T11:14:01.658747Z","last_active":"2019-10-25T11:20:32.796786818Z","online":true,"name":"Ivana"},"created_at":"2019-10-02T08:51:31.254688Z","updated_at":"2019-10-02T08:51:31.254688Z"}],"member_count":2,"config":{"created_at":"2019-09-03T10:36:23.854837Z","updated_at":"2019-10-15T12:42:50.94439Z","name":"messaging","typing_events":true,"read_events":true,"connect_events":true,"search":true,"reactions":true,"replies":true,"mutes":true,"uploads":true,"message_retention":"infinite","max_message_length":5000,"automod":"disabled","automod_behavior":"flag","commands":[{"name":"giphy","description":"Post a random gif to the channel","args":"[text]","set":"fun_set"}]}},"created_at":"2019-10-25T11:20:44.314836748Z","unread_count":23,"total_unread_count":23,"unread_channels":2}
2019-10-25 13:20:49.340 15480-15613/de.tourlina.Tourlina.staging D/WebSocketService: Received event of type NOTIFICATION_MESSAGE_NEW
```
I have an EventHandler created like this in MainActivity:
```StreamChat.getInstance(this).addEventHandler(new ChatEventHandler() {
            @Override
            public void onNotificationMessageNew(Channel channel, Event event) {
                super.onNotificationMessageNew(channel, event);
                Log.d(TAG, "New notification message is received");
            }
            @Override
            public void onMessageNew(Channel channel, Event event) {
                super.onMessageNew(channel, event);
                Log.d(TAG, "New message is received");
            }
        });
```
Expecting to recieve a callback in onNotficiationMessageNew but there is nothing. Is my implementation wrong or

**Comment by tbarbugli** [comment]
Are you using the latest version? We had a couple fixes around notification event handling which included moving `onNotificationMessageNew` from `ChatChannelEventHandler` to `ChatEventHandler`.

**Comment by savked** [comment]
Just saw that the new version was released, I was using 3.1.0-alpha1. Thank you very much. Sorry for bothering.
